src/evaluatorfrontend.h

```diff
@@ -24,6 +24,7 @@ enum ErrorCode {
 class EvaluatorFrontend {
 public:
 	bool plot = false;
+	bool specificPlotting = false;
```

Comment:
Behøver man overhovedet en ekstra variabel? Man kunne jo bare sige `if (!desiredChemicals.empty())` i stedet for `if (specificPlotting)`. Det ekskluderer muligheden for at køre programmet mens man ikke plotter noget, men det er heller ikke en mulighed i UI'en alligevel.